packages/v4/patternfly-docs/content/get-started/release-highlights.md

```diff
@@ -0,0 +1,168 @@
+
+Our components often allow you to display descriptive or status text below a form field to help users make decisions, but the implementation of this text hasn’t been consistent across components. We wanted to clean things up so that this implementation is more consistent.
+
+As a result, a few components were updated to use the same [`HelperText` component](/components/helper-text), rather than their own disparate helper-text-like implementation. This change affects the following components:
+
+- [Date picker](/get-started/upgrade#date-picker)
+- [Time picker](/get-started/upgrade#time-picker)
+- [Form](/get-started/upgrade#form) 
+    - The `FormGroup` component is especially affected, as we've removed the following helper text related properties in PatternFly 5: 
+        - `helperText`
+        - `helperTextInvalid`
+        - `validated`
+        - `helperTextIcon`
+        - `helperTextInvalidIcon`
+        - `isHelperTextBeforeField`
+
+        To add helper text to a form group in place of these properties you should use the `FormHelperText`, `HelperText`, and `HelperTextItem` components directly as part of the `FormGroup`’s children. This implementation is shown in PatternFly's [invalid form example.](components/forms/form#invalid)
+
+### Input group
+
+Previously, the input group styles were too narrowly coupled with the types of children they contained. It did not match the approach we took to styling other groups of components. This update enabled more flexibility for the types of children allowed in an input group, and simplified the style rules for things like spacing, focus management, and item size manipulation.
+
+As a result, input group items no longer automatically fill the available space of their container. To retain previous styling, new `InputGroupItem` components may need to have the `isFill`, `isBox`, and/or `isPlain` properties adjusted. This is likely necessary for `FormSelect`, `TextInput`, `Select`, and `TextArea`, which may particularly require the `isFill` property to be passed to `InputGroupItem`.
+
+## Refactored components 
+
+As PatternFly has matured, a number of its popular components have been repeatedly enhanced. Their complexity and fragility has grown to become an ever-growing maintenance problem. Rather than continuing to enhance these delicate components, as part of PatternFly 5, we have adjusted our implementation and recommendations to support new components that are more flexible, composable, and easier to maintain. As a result, we are deprecating our previous implementations in favor of our newer, more composable components.
+
+The following table outlines the components that are affected, as well as the component(s) that we recommend using as a replacement.
+
+| Component  | Recommended replacement  |
+| --- | --- |
+| [Application launcher](/components/menus/application-launcher/react-deprecated)  | [Dropdown-next](/components/menus/dropdown), [select-next](/components/menus/select)|
```

Comment:
I think we want tot mention that these components can be built with the menu.  That is how we demonstrate building them.  Maybe even link to demos.

Reply:
The application launcher's docs have two tabs now. One is the deprecated examples, and one is react-demos which show how to build an Application launcher using a Dropdown.

The same is done for Context selector and Options menu (although options menu is built using a Select, not a Dropdown). 

So I think this is good as is